observers/slack_resource.cpp

```diff
@@ -0,0 +1,82 @@
+#include <memory>
+#include <vector>
+
+#include "glog/logging.h"
+
+#include "mesos/mesos.hpp"
+
+#include "observers/slack_resource.hpp"
+
+#include "serenity/metrics_helpers.hpp"
+
+namespace mesos {
+namespace serenity {
+
+Try<Nothing> SlackResourceObserver::consume(const ResourceUsage& usage) {
+  std::unique_ptr<ExecutorSet> newSamples(new ExecutorSet());
```

Comment:
See:
- https://github.com/apache/mesos/blob/master/3rdparty/libprocess/src/process.cpp#L106
- https://github.com/apache/mesos/blob/master/src/master/master.cpp#L89

I had in mind:

```
using std::unique_ptr;
...
unique_ptr<ExecutorSet> ...
```